docs: add staking description to README.md

dsy124 opened the pull request:
- Add staking description "Staking on Layer 1 CKB"
- Also change "Native Cross-chain Communication" to "Native Cross-Chain Communication" (seen in "Developer-Friendly Design")

dsy124 committed:
Add staking description to README.md

- Add staking description "Staking on Layer 1 CKB"
- Also change "Native Cross-chain Communication" to "Native Cross-Chain Communication" (seen in "Developer-Friendly Design")

KaoImin commented on README.md:
```diff
@@ -27,10 +27,14 @@ Axon is a high-performance layer2 framework with native cross-chain function. B
 
+### Staking on Layer 1 CKB
+
+An Axon-based chain is a Proof-of-Stake (PoS) network. Native AT token holders can stake their tokens to become validators and/or delegate their tokens to other validators to earn rewards. Different from most other sidechains whose staking happens on the layer 2 level, Axon’s staking mechanism takes place on the layer 1 CKB. Validators and other participants stake their native AT tokens on CKB, w ...
```
```suggestion
An Axon-based chain is a Proof-of-Stake (PoS) network. Native AT token holders can stake their tokens to become validators and/or delegate their tokens to other validators to earn rewards. Different from most other sidechains whose staking happens on the layer 2 level, Axon’s staking mechanism takes place on the layer 1 CKB. Validators and other participants stake their native AT token which is an xUDT corresponding to the Axon-based chain on CKB, which uses a Proof-of-Work (PoW) consensus mechanism. This unique staking design helps Axon-based sidechains enjoy the highest degree of decentralization and security from the layer 1 while maintain their high performance and sovereignty as independent layer 2 networks. 
```

linnnsss replied:
An Axon-based chains is a Proof-of-Stake (PoS) network. Holders of the Native AT token—Axon't utility token—can stake their tokens to become validators or delegate their tokens to other validators in exchange for rewards. Unlike other sidechains where staking takes place on layer 2, Axon's staking is grounded on layer 1 CKB. Validators and other participants stake their native AT tokens, which  are xUDTs (Extensible User Defined Tokens). The consensus mechanism used by CKB is Proof-of-Work (PoW). This unique staking design allows Axon-based sidechains to enjoy the highest level of decentralization and security from layer 1 while maintaining their high performance and sovereignty as independent layer 2 networks.

linnnsss replied:
Comments on the above comments:
Explain "AT token" when it was introduced for the first time.
"AT token" can be defined as "utility token" that allows holders to participate in consensus (to avoid confusion with "value token").
Add the full name of xUDT.

kelisi129 commented on README.md:
```diff
@@ -27,10 +27,14 @@ Axon is a high-performance layer2 framework with native cross-chain function. B
-### Native Cross-chain Communication
+### Native Cross-Chain Communication
 
 Openness and mobility are the foundation of social development, so is blockchain. Cross-chain function enhances liquidity for the web3 ecosystem. Axon develops native cross-chain communication without any bridge. Each Axon-based chain can speak to [CKB](https://www.nervos.org), other Axon-based chains, and any [IBC](https://ibcprotocol.org) compatible chains. Axon will embed more cross-chain prot ...
```
"Openness and mobility are the foundation of social development, so is blockchain." --> openness/mobility & blockchain are two different categories, trying other analogies?

_thoughts:
Openness and mobility are the foundation of social development, and technologies like blockchain where cross-chain function facilitates interoperability drives social changes. As a new cross-chain initiative, Axon is developing xxxx._

dsy124 replied:
Agreed with some changes to the opening sentence "Openness and mobility are the foundation of social development, so is blockchain." --> I guess we are trying to convey that openness are mobility are also the foundation of blockchain? Grammatically, it is not particularly clear.

kelisi129 replied:
From my perspective, social development & blockchain dont fall into the same category either. Social development is a terminology/ theory explaining the changes within the society as a whole, yet blockchain is a technology, one of the factors that is driving the social change.

dsy124 replied:
Got it! Thank you!

kelisi129 commented on README.md:
```diff
@@ -27,10 +27,14 @@ Axon is a high-performance layer2 framework with native cross-chain function. B
 
+### Staking on Layer 1 CKB
+
+An Axon-based chain is a Proof-of-Stake (PoS) network. Native AT token holders can stake their tokens to become validators and/or delegate their tokens to other validators to earn rewards. Different from most other sidechains whose staking happens on the layer 2 level, Axon’s staking mechanism takes place on the layer 1 CKB. Validators and other participants stake their native AT tokens on CKB, w ...
```
Different from most other sidechains **where** staking happens on **layer 2**

dsy124 replied:
I think either case works. no strong preferences.

kelisi129 commented on README.md:
```diff
@@ -27,10 +27,14 @@ Axon is a high-performance layer2 framework with native cross-chain function. B
+
 ## Roadmap
 
 Most of the infrastructure has been done and some substantial features to be developed are as below:
```
Here are some substantial features we're working on:

dsy124 replied:
Sorry could you please elaborate?

kelisi129 replied:
I suggest that deleting "Most of the infrastructure has been done", focusing on what we're doing to make Axon staking mechanism more outstanding/ some highlights of the staking mechanism.

dsy124 replied:
The Roadmap is another high-level section focusing on the project as a whole and not just staking I think. But thanks for bringing up a good point. @KaoImin do we need to add anything staking-related to roadmap?

dsy124 committed:
docs: update staking description to README.md

Incorporating comments:
- explain AT tokens 
- spell out xUDT for the first time

dsy124 committed:
docs: update staking description in README

- clarify AT tokens are a general term referring to xUDTs on Axon-based app chains
- explain that AT tokens are designed/issue by the app chain's development team
- use app chains, instead of sidechains

dsy124 committed:
docs: updates to staking description 1.3 in README

- app chain to appchain
- specify that xUDTs are issued on CKB ("released on CKB").

dsy124 committed:
docs: update staking description 1.4

- changed "enables" to "requires"
- uniform spelling: Layer 1 and Layer 2
